#
# Copyright 2007 Sun Microsystems, Inc.  All rights reserved.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the
# "Software"), to deal in the Software without restriction, including
# without limitation the rights to use, copy, modify, merge, publish,
# distribute, and/or sell copies of the Software, and to permit persons
# to whom the Software is furnished to do so, provided that the above
# copyright notice(s) and this permission notice appear in all copies of
# the Software and that both the above copyright notice(s) and this
# permission notice appear in supporting documentation.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
# OF THIRD PARTY RIGHTS. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
# HOLDERS INCLUDED IN THIS NOTICE BE LIABLE FOR ANY CLAIM, OR ANY SPECIAL
# INDIRECT OR CONSEQUENTIAL DAMAGES, OR ANY DAMAGES WHATSOEVER RESULTING
# FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
# NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION
# WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
#
# Except as contained in this notice, the name of a copyright holder
# shall not be used in advertising or otherwise to promote the sale, use
# or other dealings in this Software without prior written authorization
# of the copyright holder.
# 
#
#

Solaris already has the old XPanoramiX* API in libXext, so we don't export
it from libXinerama to avoid symbol clashes.

--- src/Xinerama.c	2007-03-01 15:41:16.000000000 -0800
+++ src/Xinerama.c	2007-03-07 14:50:49.899520000 -0800
@@ -76,6 +76,9 @@
  *                                                                          *
  ****************************************************************************/
 
+#ifdef SUNSOFT
+static inline
+#endif /* SUNSOFT */
 Bool XPanoramiXQueryExtension (
     Display *dpy,
     int *event_basep,
@@ -94,6 +97,9 @@
 }
 
 
+#ifdef SUNSOFT
+static inline
+#endif /* SUNSOFT */
 Status XPanoramiXQueryVersion(
     Display *dpy,
     int	    *major_versionp, 
@@ -124,6 +130,7 @@
     return 1;
 }
 
+#ifndef SUNSOFT
 XPanoramiXInfo *XPanoramiXAllocInfo(void)
 {
 	return (XPanoramiXInfo *) Xmalloc (sizeof (XPanoramiXInfo));
@@ -219,6 +226,7 @@
     panoramiX_info->height = rep.height;
     return 1;
 }
+#endif /* SUNSOFT */
 
 /*******************************************************************\
   Alternate interface to make up for shortcomings in the original,
